src/store/holochain.js

```diff
@@ -285,14 +301,16 @@ export default {
       state.reconnectingIn = -1
       state.firstConnect = false
     },
-    setHoloClientAndDnaAlias (state, payload) {
-      const { holoClient, dnaAlias } = payload
-      console.log('HOLO DNA ALIAS : ', dnaAlias)
-      state.dnaAlias = dnaAlias
-      state.holoClient = holoClient
```

Comment:
The reason these were paired is because you can't make zome calls without the DNA alias being set. Will you add a check to `callZome.js` for unset dnaAlias if you're going to decoople these?

Reply:
Good reminder.  Will do.